### ISSUE TITLE
Includes username/login info in the TLD

### ISSUE THREAD

**Issue by clokep** [barseghyanartur/tld#18]
First, thanks for making this package! It's solved a few problems for me. 👍 

I'm not sure if this is meant to be handled by this package or not, but I have a couple of domains I've tried to parse with this that are in the form of `http://foo@bar.com`, which I parse as:

```python
url = 'http://foo@bar.com'
res = get_tld(url, as_object=True, fix_protocol=True, fail_silently=True)
domain = res.tld
```

But `domain` will equal `foo@bar.com` here, I'd expect just `bar.com`.

Right now I'm just splitting on `@` if it exists and handling it, which is easy, but it seems weird to have a property called `domain` that includes this information. Feel free to close as out of scope if you want!

**Comment by clokep** [comment]
I should also mention that this *sometimes* ends up as part of the subdomain, e.g. in the case of `'http://foo@www.bar.com`.

**Comment by barseghyanartur** [comment]
@clokep:

Thanks for reporting this. Sounds reasonable. Added to the TODOs.

**Comment by Yomguithereal** [comment]
PR opened.

**Comment by barseghyanartur** [comment]
Solved in branch [v0.8](https://github.com/barseghyanartur/tld/tree/v0.8). To be released in 0.8.

**Comment by barseghyanartur** [comment]
Released in 0.8.
